22:46:08,330 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:08,366 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:08,376 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:08,385 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-13 to job manager
22:46:08,450 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:08,450 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:08,451 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:09,541 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:09,561 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:09,570 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:09,570 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,423 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (8/25)
22:46:46,426 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (8/25)
22:46:49,154 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (59/200)
22:46:49,155 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (59/200)
22:46:49,157 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (59/200)
22:46:49,158 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (59/200)
22:46:50,355 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (64/200)
22:46:50,355 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (64/200)
22:46:50,357 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (64/200)
22:46:50,359 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (64/200)
22:46:50,532 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (57/200)
22:46:50,533 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (57/200)
22:46:50,534 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (57/200)
22:46:50,535 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (57/200)
22:46:50,903 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (58/200)
22:46:50,904 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (58/200)
22:46:50,905 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (58/200)
22:46:50,907 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (58/200)
22:46:51,273 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (63/200)
22:46:51,274 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (63/200)
22:46:51,275 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (63/200)
22:46:51,277 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (63/200)
22:46:51,303 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (60/200)
22:46:51,303 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (60/200)
22:46:51,305 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (60/200)
22:46:51,307 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (60/200)
22:46:52,205 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (61/200)
22:46:52,205 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (61/200)
22:46:52,207 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (61/200)
22:46:52,209 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (61/200)
22:46:52,500 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (62/200)
22:46:52,501 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (62/200)
22:46:52,502 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (62/200)
22:46:52,503 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (62/200)
22:47:20,583 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (8/25)
22:47:20,583 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (8/25)
22:49:24,503 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2002202738669431328_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54325 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:24,504 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2002202738669431328_1722 bad datanode[0] 130.149.21.17:45020
22:49:24,505 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2002202738669431328_1722 in pipeline 130.149.21.17:45020, 130.149.21.20:45020, 130.149.21.61:45020: bad datanode 130.149.21.17:45020
22:49:25,489 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7488572230347183718_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54339 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:25,490 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7488572230347183718_1722 bad datanode[0] 130.149.21.17:45020
22:49:25,491 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7488572230347183718_1722 in pipeline 130.149.21.17:45020, 130.149.21.22:45020, 130.149.21.33:45020: bad datanode 130.149.21.17:45020
22:49:29,603 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8369779653374792749_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54329 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:29,605 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8369779653374792749_1722 bad datanode[0] 130.149.21.17:45020
22:49:29,605 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8369779653374792749_1722 in pipeline 130.149.21.17:45020, 130.149.21.27:45020, 130.149.21.64:45020: bad datanode 130.149.21.17:45020
22:49:49,635 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_3988980374315155657_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54331 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:49,636 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3988980374315155657_1722 bad datanode[0] 130.149.21.17:45020
22:49:49,637 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3988980374315155657_1722 in pipeline 130.149.21.17:45020, 130.149.21.19:45020, 130.149.21.20:45020: bad datanode 130.149.21.17:45020
22:49:51,216 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7007254299472023632_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54337 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:51,217 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7007254299472023632_1722 bad datanode[0] 130.149.21.17:45020
22:49:51,218 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7007254299472023632_1722 in pipeline 130.149.21.17:45020, 130.149.21.23:45020, 130.149.21.30:45020: bad datanode 130.149.21.17:45020
22:49:52,751 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7982837751519499259_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54323 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:52,753 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7982837751519499259_1722 bad datanode[0] 130.149.21.17:45020
22:49:52,753 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7982837751519499259_1722 in pipeline 130.149.21.17:45020, 130.149.21.29:45020, 130.149.21.30:45020: bad datanode 130.149.21.17:45020
22:49:58,710 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8175748107482112301_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54321 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:58,712 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8175748107482112301_1722 bad datanode[0] 130.149.21.17:45020
22:49:58,712 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8175748107482112301_1722 in pipeline 130.149.21.17:45020, 130.149.21.23:45020, 130.149.21.33:45020: bad datanode 130.149.21.17:45020
22:50:16,898 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8766097751368340176_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:54327 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:16,899 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8766097751368340176_1722 bad datanode[0] 130.149.21.17:45020
22:50:16,900 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8766097751368340176_1722 in pipeline 130.149.21.17:45020, 130.149.21.16:45020, 130.149.21.60:45020: bad datanode 130.149.21.17:45020
22:50:30,618 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2002202738669431328_1735java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:48628 remote=/130.149.21.20:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:30,620 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2002202738669431328_1735 bad datanode[0] 130.149.21.20:45020
22:50:30,620 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2002202738669431328_1735 in pipeline 130.149.21.20:45020, 130.149.21.61:45020: bad datanode 130.149.21.20:45020
22:50:31,589 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7488572230347183718_1744java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:49544 remote=/130.149.21.22:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:31,591 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7488572230347183718_1744 bad datanode[0] 130.149.21.22:45020
22:50:31,591 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7488572230347183718_1744 in pipeline 130.149.21.22:45020, 130.149.21.33:45020: bad datanode 130.149.21.22:45020
22:50:35,703 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8369779653374792749_1766java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:35294 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:35,705 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8369779653374792749_1766 bad datanode[0] 130.149.21.27:45020
22:50:35,705 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8369779653374792749_1766 in pipeline 130.149.21.27:45020, 130.149.21.64:45020: bad datanode 130.149.21.27:45020
22:50:55,739 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_3988980374315155657_1799java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:55707 remote=/130.149.21.19:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:55,741 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3988980374315155657_1799 bad datanode[0] 130.149.21.19:45020
22:50:55,742 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3988980374315155657_1799 in pipeline 130.149.21.19:45020, 130.149.21.20:45020: bad datanode 130.149.21.19:45020
22:50:57,316 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7007254299472023632_1804java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:34579 remote=/130.149.21.23:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:57,317 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7007254299472023632_1804 bad datanode[0] 130.149.21.23:45020
22:50:57,318 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7007254299472023632_1804 in pipeline 130.149.21.23:45020, 130.149.21.30:45020: bad datanode 130.149.21.23:45020
22:50:58,895 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-7982837751519499259_1810java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:58918 remote=/130.149.21.29:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:58,896 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7982837751519499259_1810 bad datanode[0] 130.149.21.29:45020
22:50:58,897 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-7982837751519499259_1810 in pipeline 130.149.21.29:45020, 130.149.21.30:45020: bad datanode 130.149.21.29:45020
22:51:04,813 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8175748107482112301_1819java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:34585 remote=/130.149.21.23:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:04,814 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8175748107482112301_1819 bad datanode[0] 130.149.21.23:45020
22:51:04,814 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8175748107482112301_1819 in pipeline 130.149.21.23:45020, 130.149.21.33:45020: bad datanode 130.149.21.23:45020
22:51:22,998 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_8766097751368340176_1877java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.17:43547 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:23,000 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8766097751368340176_1877 bad datanode[0] 130.149.21.16:45020
22:51:23,001 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_8766097751368340176_1877 in pipeline 130.149.21.16:45020, 130.149.21.60:45020: bad datanode 130.149.21.16:45020
